### ISSUE TITLE
how to pin to taskbar?

### ISSUE THREAD

**Issue by tigros** [KrispyCamel4u/SysMonTask#103]
normally, you right click the running app on taskbar and pin it. Seems there's no pin for sysmontask.

**Comment by KrispyCamel4u** [comment]
I guess what you are looking for is "add to favourites".

**Comment by tigros** [comment]
Yes Sir, in Mint they call it pin.. I just tried in Ubuntu and add to favorites doesn't show on right click.

was just looking to replace bland gnome-system-monitor. Impressed python can handle it.  In Windows I use Process Hacker, now that's a task manager :)

Thanks!

**Comment by KrispyCamel4u** [comment]
On which version of Ubuntu you have tried it on? 
I am using PopOS and I can add it my favourites.
On mint I will verify the issue and will see about it.

**Comment by tigros** [comment]
In VMware that would be 21.10.

**Comment by tigros** [comment]
I just tried in PopOS and it's same problem. I think it's because i'm installing from source, have not tried 'sudo apt install sysmontask' at all.

**Comment by Danny3** [comment]
On Kubuntu 21.10 with latest KDE Plasma it works, it can be pinned both from the start menu and by right-clicking on the icon after it has been opened.

**Comment by tigros** [comment]
confirmed, it does work fine in Kubuntu even when installing from source.
